### ISSUE TITLE
Is it possible to run a game through the py file

### ISSUE THREAD

**Issue by heeeeeeeeeeh** [renpy/renpy#5430]
Is it possible to link modules like renpy and pygame_sdl2 from the game folder to the system's python and run the game . I'm using a school laptop and they don't allow running .exe files, but for some reason nuget packages seem to run fine. I downloaded python3.9 from [here](https://www.nuget.org/packages/python/3.9.13), and it runs.  This leaves me with running game through the py file. I tried compiling pygame_sdl2 and renpy, but i get this error error LNK2001: unresolved external symbol SDL_JoystickGetDeviceInstanceID.

**Comment by bedwardly-down** [comment]
Are you able to run a local Virtualenv using https://docs.python.org/3/library/venv.html ? You can use that to create a “jail” for your games with its own python version that’s separate from the system files

**Comment by renpytom** [comment]
Not easily, unfortunately.  You may needed a newer (or different) version of SDL2 to build pygame_sdl2.

**Comment by heeeeeeeeeeh** [comment]
> Not easily, unfortunately. You may needed a newer (or different) version of SDL2 to build pygame_sdl2.
I did get newer version of SDL2 and built pygame_sdl2, but I'am having trouble building renpy.  When compiling renpy it asks for a fribidi.lib file. I compiled fribidi in the MSYS environment but it only produces .a static libraries. I tried converting the .a file to .lib with the command `lib \machine:x64 libfribidi.a` build i get this error:  `renpybidicore.obj : error LNK2001: unresolved external symbol __imp_fribidi_log2vis
build\lib.win-amd64-3.9\_renpybidi.cp39-win_amd64.pyd : fatal error LNK1120: 1 unresolved externals` Is there a way around this?
